foxy/distribution.yaml

```diff
@@ -1100,15 +1100,12 @@ repositories:
     doc:
       type: git
       url: https://github.com/kobuki-base/kobuki_core.git
-      version: release/1.1.x
+      version: release/1.2.x
     release:
-      packages:
-      - kobuki_dock_drive
-      - kobuki_driver
```

Comment:
This will result in these two packages to be removed. I just wanted to check that this is intentional?

Reply:
@stonier friendly ping.

Reply:
Yes, intentional - it's been collapsed to a single package (kobuki_core).